src/ui/DynamicFacet/DynamicFacetValues/DynamicFacetValue.ts

```diff
@@ -87,15 +87,15 @@ export class DynamicFacetValue implements IDynamicFacetValue {
     const selectOrUnselect = !this.isSelected ? 'SelectValueWithResultCount' : 'UnselectValueWithResultCount';
     const resultCount = l('ResultCount', this.formattedCount);
 
-    return `${l(selectOrUnselect, this.valueCaption, resultCount)}`;
+    return `${l(selectOrUnselect, this.displayValue, resultCount)}`;
```

Comment:
Shouldn't the aria-label contain the value caption if it is specified?

Reply:
the displayValue will always be the right value (see previous explanation)